modules/check_in/config/initializers/statsd.rb

```diff
@@ -61,8 +61,10 @@ def check_in_type(params)
 
   def metric_prefix(headers, params)
     check_in_param = params[:checkInType] || params.dig(:session, :check_in_type)
+    facility_type_param = params[:facilityType] || params.dig(:session, :facility_type)
 
     prefix = check_in_param == 'preCheckIn' ? 'pre_check_in' : 'check_in'
+    prefix = facility_type_param unless facility_type_param.nil?
```

Comment:
Thank you, the changes look very simple. Since facility_type will be a query parameter, we can reduce it to a single line.